docs/html/user_guide.rst

```diff
@@ -440,6 +440,15 @@ and ``--no-cache-dir``, falsy values have to be used:
     no-compile = no
     no-warn-script-location = false
 
+For options which can be repeated like ``--verbose`` and ``--quite``,
+a nonnegative integer could be used to represent the level to be specified:
```

Comment:
```suggestion
a non-negative integer can be used to represent the level to be specified:
```

Reply:
`nonnegative` -> `non-negative` can be a personal choice. I just think the latter is more readable.